bazel/repository_locations.bzl

```diff
@@ -510,20 +510,36 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-07-28",
         cpe = "cpe:2.3:a:libevent_project:libevent:*",
     ),
+    net_colm_open_source_colm = dict(
+        project_name = "Colm",
+        project_desc = "The Colm Programming Language",
+        project_url = "https://www.colm.net/open-source/colm/",
+        version = "2d8ba76ddaf6634f285d0a81ee42d5ee77d084cf",
+        sha256 = "0399e9bef7603a8f3d94acd0b0af6b5944cc3103e586734719379d3ec09620c0",
+        strip_prefix = "colm-{version}",
+        urls = ["https://github.com/adrian-thurston/colm/archive/{version}.tar.gz"],
+        use_category = ["dataplane_ext"],
+        extensions = [
+            "envoy.matching.input_matchers.hyperscan",
+            "envoy.regex_engines.hyperscan",
+        ],
+        release_date = "2021-12-28",
+        cpe = "N/A",
+    ),
     net_colm_open_source_ragel = dict(
         project_name = "Ragel",
         project_desc = "Ragel State Machine Compiler",
         project_url = "https://www.colm.net/open-source/ragel/",
-        version = "6.10",
-        sha256 = "5f156edb65d20b856d638dd9ee2dfb43285914d9aa2b6ec779dac0270cd56c3f",
+        version = "d4577c924451b331c73c8ed0af04f6efd35ac0b4",
+        sha256 = "fa3474d50da9c870b79b51ad43f8d11cdf05268f5ec05a602ecd5b1b5f5febb0",
```

Comment:
Can you add a comment here on why we are using this SHA vs. the release version (just about the license switch with perhaps a link to the page describing the license of this thing). Thank you.

/wait

Reply:
> SHA vs. the release version

The release version contains a bug to prevent building static libraries. So we have to use the latest one. I will document them, thanks.